2.5/test/run

```diff
@@ -214,13 +214,24 @@ function test_scl_variables_in_dockerfile() {
  fi
 }
 
+function test_from_dockerfile() {
+  rootdir=$(readlink -zf ${image_dir}/..)
+  dockerfile="Dockerfile${1:-}"
+  info "Check building using a $dockerfile"
+  ct_test_app_dockerfile $rootdir/examples/from-dockerfile/$dockerfile 'https://github.com/sclorg/rails-ex.git' 'Welcome to your Rails application on OpenShift' app-src
```

Comment:
This is actually always Ruby 2.7, right?

Any response to https://github.com/sclorg/s2i-ruby-container/pull/295#discussion_r535682327 ?

Reply:
Yes, from my point of view it's enough to test it only once